seacrowd/sea_datasets/xstorycloze/xstorycloze.py

```diff
@@ -0,0 +1,177 @@
+import csv
+import itertools
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import SCHEMA_TO_FEATURES, Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{lin2022fewshot,
+    author       = {Xi Victoria Lin and
+                    Todor Mihaylov and
+                    Mikel Artetxe and
+                    Tianlu Wang and
+                    Shuohui Chen and
+                    Daniel Simig and
+                    Myle Ott and
+                    Naman Goyal and
+                    Shruti Bhosale and
+                    Jingfei Du and
+                    Ramakanth Pasunuru and
+                    Sam Shleifer and
+                    Punit Singh Koura and
+                    Vishrav Chaudhary and
+                    Brian O'Horo and
+                    Jeff Wang and
+                    Luke Zettlemoyer and
+                    Zornitsa Kozareva and
+                    Mona T. Diab and
+                    Veselin Stoyanov and
+                    Xian Li},
+    editor       = {Yoav Goldberg and
+                    Zornitsa Kozareva and
+                    Yue Zhang},
+    title        = {Few-shot Learning with Multilingual Generative Language Models},
+    booktitle    = {Proceedings of the 2022 Conference on Empirical Methods in Natural
+                    Language Processing, {EMNLP} 2022, Abu Dhabi, United Arab Emirates,
+                    December 7-11, 2022},
+    pages        = {9019--9052},
+    publisher    = {Association for Computational Linguistics},
+    year         = {2022},
+    url          = {https://doi.org/10.18653/v1/2022.emnlp-main.616},
+    doi          = {10.18653/V1/2022.EMNLP-MAIN.616},
+}
+"""
+
+_DATASETNAME = "xstorycloze"
+_DESCRIPTION = """\
+XStoryCloze consists of the professionally translated version of the English StoryCloze
+dataset (Spring 2016 version) to 10 non-English languages. This dataset is released by
+Meta AI.
+"""
+_HOMEPAGE = "https://huggingface.co/datasets/juletxara/xstory_cloze"
+_LANGUAGES = ["ind", "mya"]
+_LICENSE = Licenses.CC_BY_SA_4_0.value
+
+_LOCAL = False
+_BASE_URL = "https://huggingface.co/datasets/juletxara/xstory_cloze/resolve/main/spring2016.val.{lang}.tsv.split_20_80_{split}.tsv"
+_SUPPORTED_TASKS = [Tasks.COMMONSENSE_REASONING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class xStoryClozeDataset(datasets.GeneratorBasedBuilder):
```

Comment:
use PascalCase (first letter is capitalized too)
```suggestion
class XStoryClozeDataset(datasets.GeneratorBasedBuilder):
```